tests/index.test.ts

```diff
@@ -141,7 +154,7 @@ describe('ParaSwap SDK', () => {
   });
 
   test('Get_Spender', async () => {
-    const spender = await paraSwap.getTokenTransferProxy();
+    const spender = await paraSwap.getSpender();
     expect(provider.utils.isAddress(spender as string));
```

Comment:
Would be more useful to assert the type than cast
```ts
// it would act same as `expect`
assert(typeof spender === 'string', 'spender must be valid');
// spender is now string
```
Or even use a [library](https://github.com/unional/type-plus#type-assertion) to check types specifically. Which could be a nice addition later, to test types